### PR TITLE
Hw5

### PR DESCRIPTION
**Pull request by MSt1ch**

### CODE REVIEW

File: src/error-hander/errorHandlers.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {NextFunction, Request, Response} from 'express';
+import {INTERNAL_SERVER_ERROR} from 'http-status';
+import {logger} from '../utils/logger';
+
+export const errorHandlerMiddleware = (err: Error, req: Request, res: Response, next: NextFunction) => {
+  logger.error(JSON.stringify({errorMessage: err.message}));
+  res.status(INTERNAL_SERVER_ERROR).json({error: 'Internal Service Error'});
+  next();
```
Comment (aliaksandr-shyshonak) [inline src/error-hander/errorHandlers.ts:8]: Just wondering, why `next` is needed since the response has been sent already?

File: src/index.ts [hunk 2]
```diff
@@ -13,18 +18,31 @@ if (!process.env.PORT) {
...
 const app = express();
 
 app.listen(PORT, () => {
-  console.log(`Listening on port ${PORT}`);
+  logger(`Listening on port ${PORT}`);
 });
 
 app.set('case sensitive routing', false);
+
+app.use(bodyParser.json());
```
Comment (aliaksandr-shyshonak) [inline src/index.ts:35]: Just wondering, why have you added `bodyParser`?

**Review by aliaksandr-shyshonak — APPROVED** [review]

File: src/index.ts (hunk 2 above)
Comment (MSt1ch) [reply src/index.ts:35]: As I understand this plugin helps to convert request body to valid JSON format
You can read the documentation below: 
https://www.npmjs.com/package/body-parser

File: src/error-hander/errorHandlers.ts (hunk 1 above)
Comment (MSt1ch) [reply src/error-hander/errorHandlers.ts:8]: Hm, you are absolutely right, it looks unnecessary at this.

File: src/index.ts (hunk 2 above)
Comment (aliaksandr-shyshonak) [reply src/index.ts:35]: But doesn't it do the same as `express.json()` at line 40?

Comment (MSt1ch) [reply src/index.ts:35]: Hm, yeah thank you. This line is unnecessary.

**Pull request merged**